crawl-ref/source/fontwrapper-ft.cc

```diff
@@ -86,12 +104,11 @@ bool FTFontWrapper::load_font(const char *font_name, unsigned int font_size,
     if (!f)
         die_noline("Could not read font '%s'\n", font_name);
     unsigned long size = file_size(f);
-    FT_Byte *ttf = (FT_Byte*)malloc(size);
+    ttf = (FT_Byte*)malloc(size);
```

Comment:
I realize you're just editing code that was already there, but while you're in there I wonder if this malloc/free pair could be converted to new/delete?  There may be some arcane reason why not that I'm missing since I don't know much about freetype (though FT_Byte is probably just unsigned char), but I think we try to avoid malloc.  Especially given that ttf is now a member variable.